src/app/admin/components/system-configuration/event-mappings/delete-event-mapping/delete-event-mapping.component.ts

```diff
@@ -0,0 +1,57 @@
+import { CommonModule } from '@angular/common';
+import { Component, inject } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DataTableComponent } from '@common/data-table/data-table.component';
+import { DeleteComponent } from '@common/delete/delete.component';
+import { GovukHeadingComponent } from '@common/govuk-heading/govuk-heading.component';
+import { DatatableColumn } from '@core-types/index';
+import { TableRowTemplateDirective } from '@directives/table-row-template.directive';
+import { LuxonDatePipe } from '@pipes/luxon-date.pipe';
+import { EventMappingsService } from '@services/event-mappings/event-mappings.service';
+
+@Component({
+  selector: 'app-delete-event-mapping',
+  standalone: true,
+  templateUrl: './delete-event-mapping.component.html',
+  styleUrl: './delete-event-mapping.component.scss',
+  imports: [
+    DeleteComponent,
+    DataTableComponent,
+    CommonModule,
+    LuxonDatePipe,
+    TableRowTemplateDirective,
+    GovukHeadingComponent,
+  ],
+})
+export class DeleteEventMappingComponent {
+  eventMappingsService = inject(EventMappingsService);
+  route = inject(ActivatedRoute);
+  router = inject(Router);
+
+  id = +this.route.snapshot.params.id;
```

Comment:
Should the `+` be there?

Reply:
The + unary operator will convert the query params from string to the expected number type

Reply:
Didn't know that trick, nice